openbb_platform/providers/bis/openbb_bis/utils/constants.py

```diff
@@ -0,0 +1,103 @@
+"""Constants for the BIS provider."""
+
+
+
+COUNTRY_TO_CODE_HOUSE_PRICE_INDEX = {
```

Comment:
For the normalized index - nominal and real - there are also regional codes for items like "Advanced Economies". I believe they follow the same code structure as OECD but may have their own aggregates as well, like "XW" for world. We also need a parameter state to represent "all", every available reference area for the series.